lab_6_pipeline/pipeline.py

```diff
@@ -19,10 +43,22 @@ def __init__(self, path_to_raw_txt_data: Path):
         Initializes CorpusManager
         """
 
+        self._path = path_to_raw_txt_data
+        self._storage = []
```

Comment:
Are you sure it is supposed to be a list?

Reply:
fixed